quickstarts/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc

```diff
@@ -0,0 +1,111 @@
+= Add ee-security to the EAP quickstarts
+:author:            Farah Juma
+:email:             fjuma@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The ee-security quickstart is currently a community-only quickstart because it's not part of the EAP quickstarts repo.
+Nothing else demonstrates how to configure EE security so this should be added to the EAP quickstarts as well.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/EAP7-1916[EAP7-1916]
+
+=== Related Issues
+
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [ ] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
```

Comment:
In README of quickstart is isn't mentioned managed domain, bootable jar. And OpenShift is explicitly mentioned as not compatible for this QS.

Should be update the AD or README?